### DISCUSSION TITLE
Why all data in second file "199801020030.LDASOUT_DOMAIN1",what's the use of other files?I don't understand it.

### DISCUSSION THREAD

**Discussion by anlei-code** [NCAR/hrldas#226] [Q&A] [answered]
<img width="363" alt="05173d3eb9051a6a20b9022742d6e20" src="https://github.com/user-attachments/assets/2cffe215-6c33-4db1-8ee6-e5eaeeddbc22">

**Comment by cenlinhe** [comment]
The first 00Z time output file includes only the values for those key state variables with their initial values read from your HRLDAS_setup file and initialization module setup. That is why you see a lot of -9999 or NaN or 0 values in all diagnostic variables. Typically, the first 00Z time output file will not be used.

**Reply by anlei-code** [reply]
![image](https://github.com/user-attachments/assets/3b403934-3b81-41c7-b4f2-6b0408d07505)
![image](https://github.com/user-attachments/assets/72df40c7-3c8e-4ad9-8851-d56e6292401e)
just one file like the second image,the most files like the first image.SO,I think the all output variables in one file?and other file just current value？

**Reply by cenlinhe** [reply]
This is weird. I never saw this issue. The second file with all the variables should be the correct one. For all files, it should be like the second one with a full set of the output files. I am not sure why your first file only includes 7 output files. Did you see any errors when the model generates the first file?
Also, could you please copy and paste your namelist.hrldas content here? I want to see how you set up the model output timestep and files.

**Reply by anlei-code** [reply]
I know how to set SPLIT_OUTPUT_COUNT and RESTART_FREQUENCY_HOURS,thank you for your help ,I know the modle better!!

**Comment by anlei-code** [comment]
[namelist.txt](https://github.com/user-attachments/files/17686648/namelist.txt)
thank you for help,I'm the freshman for the research,so it would be a difficulty  for me.But I will try my best to do.thank you for your help again

**Reply by anlei-code** [reply, marked as answer]
I know how to set SPLIT_OUTPUT_COUNT and RESTART_FREQUENCY_HOURS,thank you for your help ,I know the modle better!!

**Reply by cenlinhe** [reply]
great. Your previous output file issue is indeed caused by your namelist setup. You can set up  SPLIT_OUTPUT_COUNT = 1
